helm/flink-kubernetes-operator/templates/flink-operator.yaml

```diff
@@ -34,6 +34,12 @@ spec:
         {{- include "flink-operator.selectorLabels" . | nindent 8 }}
       annotations:
         kubectl.kubernetes.io/default-container: {{ .Chart.Name }}
+      {{- $keyExist := .Values.operatorPodDeployment | default dict -}}
```

Comment:
This could be just `if .Values.operatorPodDeployment.annotations`, right?
We are using the same convention throughout the helm chart.

Reply:
How about using a construct like this?

```
{{- if index (.Values.defaultConfiguration) "flink-conf.yaml" }}
  {{- index (.Values.defaultConfiguration) "flink-conf.yaml" | nindent 4 -}}
{{- end }}
```